packages/front-end/components/Experiment/Results.tsx

```diff
@@ -158,9 +162,9 @@ const Results: FC<{
   const showBreakDownResults =
     !draftMode &&
     hasData &&
-    snapshot?.dimension &&
-    snapshot.dimension.substring(0, 8) !== "pre:date" && // todo: refactor hardcoded dimension
-    analysis?.settings?.dimensions?.length; // todo: needed? separate desired vs actual
+    ((snapshot?.dimension &&
+      snapshot.dimension.substring(0, 8) !== "pre:date") ||
+      analysis?.settings?.dimensions?.length);
```

Comment:
suggestion: If we do it like this we change the type of `showBreakDownResults` from `number | boolean | undefined` to only `boolean`.

```suggestion
    ((snapshot?.dimension &&
      snapshot.dimension.substring(0, 8) !== "pre:date") ||
      (analysis?.settings?.dimensions?.length ?? 0) > 0);
```